Mathlib/Algebra/Category/GroupCat/Basic.lean

```diff
@@ -494,7 +494,9 @@ end CategoryTheory.Aut
 instance GroupCat.forget_reflects_isos : ReflectsIsomorphisms (forget GroupCat.{u}) where
   reflects {X Y} f _ := by
     let i := asIso ((forget GroupCat).map f)
-    let e : X ≃* Y := { i.toEquiv with map_mul' := by aesop }
+    let e : X ≃* Y :=
+      { i.toEquiv with
+        map_mul' := by intros; dsimp; erw [Iso.toEquiv_fun, asIso_hom, map_mul] }
```

Comment:
Maybe a comment here saying it used to be by `aesop`? I'd like to not just let proofs rot like this.